executor/builder.go

```diff
@@ -1913,6 +1913,8 @@ func (b *executorBuilder) getSnapshot() (kv.Snapshot, error) {
 	replicaReadType := sessVars.GetReplicaRead()
 	snapshot.SetOption(kv.ReadReplicaScope, b.readReplicaScope)
 	snapshot.SetOption(kv.TaskID, sessVars.StmtCtx.TaskID)
+	snapshot.SetOption(kv.ResourceGroupName, sessVars.ResourceGroupName)
```

Comment:
Seems our test about `oltp_point_select` works, does that not pass here?

Reply:
I move this line from buildPointGet/buildBatchPointGet here ,the logic is not change. Just simplify the logic a bit.